examples/helia-101/101-basics.js

```diff
@@ -1,39 +1,60 @@
 /* eslint-disable no-console */
+// @ts-check
 
+import * as nodefs from 'fs'
+import { devNull } from 'node:os'
+import { pipeline } from 'stream/promises'
 import { createHeliaHTTP } from '@helia/http'
-import { unixfs } from '@helia/unixfs'
+import { unixfs, urlSource } from '@helia/unixfs'
 
-// create a Helia node
+// `@helia/http` is an light http-only version Helia with the same API,
+// which is useful for simple use cases, where you don't need p2p networking to provide data to other nodes.
+// Since this example is focused on UnixFS without p2p networking, we can use the `@helia/http` package.
 const helia = await createHeliaHTTP()
 
-// create a filesystem on top of Helia, in this case it's UnixFS
+// UnixFS allows you to encode files and directories such that they are addressed by CIDs and can be retrieved by other nodes on the network
 const fs = unixfs(helia)
 
-// add a file and wrap in a directory
-const readmeCid = await fs.addFile({
-  path: './README.md'
-}, {
-  wrapWithDirectory: true
-})
-
-console.log('Added README.md file:', readmeCid.toString())
-
-// we will use this TextEncoder to turn strings into Uint8Arrays
+// we will use this TextEncoder to turn strings into Uint8Arrays which we can add to the node
 const encoder = new TextEncoder()
 
-// add the bytes to your node and receive a unique content identifier
+// addBytes takes raw bytes and returns a raw block CID for the content
```

Comment:
nit: reader may assume they will always get a raw block, maybe note that wbig files will be chunked and CID will no longer be for raw block, but for daf-pb root of a bigger DAG?

```suggestion
// addBytes takes raw bytes and returns a raw block CID for the content
// (a bigger file would be chunked and CID would be for dag-pb block instead)
```

Reply:
```suggestion
// addBytes takes raw bytes and returns a raw block CID for the content
// (larger (over 1 MiB) binary arrays are chunked and return a dag-pb block CID)
```